internal/types/kruizePayload/common.go

```diff
@@ -37,37 +37,47 @@ type aggregation_info struct {
 	Format string `json:"format,omitempty"`
 }
 
-type recommendation struct {
-	Data          map[string]recommendationType `json:"data,omitempty"`
-	Notifications map[string]notification       `json:"notifications,omitempty"`
+type Recommendation struct {
+	Version       string                        `json:"version,omitempty"`
+	Data          map[string]RecommendationData `json:"data,omitempty"`
+	Notifications map[string]Notification       `json:"notifications,omitempty"`
 }
 
-type notification struct {
+type Notification struct {
 	NotifyType string `json:"type,omitempty"`
 	Message    string `json:"message,omitempty"`
 	Code       int    `json:"code,omitempty"`
 }
 
-type recommendationType struct {
-	Duration_based termbased `json:"duration_based,omitempty"`
+type RecommendationEngineObject struct {
```

Comment:
Not in this pull request, however if we can separate out types in separate files then that would be awesome! I mean, a file for recommendation related times etc.

Reply:
We do have that, https://github.com/RedHatInsights/ros-ocp-backend/blob/main/internal/model/recommendation_set.go
The highlighted part of the code is just to consume the json from Kruize